### PR TITLE
Redshift Connector

### PR DESCRIPTION
**Pull request by kustosz**
### Pull Request Description
Adds DB connector for redshift.

### Important Notes
This is not automatically tested. A ticket for tests is #1984 
### Checklist

Please include the following checklist in your PR:

- [x] The documentation has been updated if necessary.
- [x] All code conforms to the [Scala](https://github.com/enso-org/enso/blob/main/docs/style-guide/scala.md), [Java](https://github.com/enso-org/enso/blob/main/docs/style-guide/java.md), and [Rust](https://github.com/enso-org/enso/blob/main/docs/style-guide/rust.md) style guides.
- [x] All documentation and configuration conforms to the [markdown](https://github.com/enso-org/enso/blob/main/docs/style-guide/markdown.md) and [YAML](https://github.com/enso-org/enso/blob/main/docs/style-guide/yaml.md) style guides.
- [x] All code has been tested where possible.

### CODE REVIEW

File: distribution/lib/Standard/Database/0.2.28-SNAPSHOT/THIRD-PARTY/org.apache.httpcomponents.httpclient-4.5.13/NOTICES [hunk 1]
```diff
@@ -0,0 +1 @@
+regarding copyright ownership.  The ASF licenses this file
```
Comment (radeusgd) [inline distribution/lib/Standard/Database/0.2.28-SNAPSHOT/THIRD-PARTY/org.apache.httpcomponents.httpclient-4.5.13/NOTICES]: This is wrong. It is some random line out of context.

It was probably saved using "Keep" while "Keep with context" should have been used. In some situations "Keep in context" cannot be used, so you need to do "Ignore" and manually add the text including all relevant context to `copyright-add` file. As explained in the license review process docs: 
image

File: distribution/lib/Standard/Database/0.2.28-SNAPSHOT/THIRD-PARTY/org.apache.httpcomponents.httpcore-4.4.13/NOTICES [hunk 2]
```diff
@@ -0,0 +1 @@
+regarding copyright ownership.  The ASF licenses this file
```
Comment (radeusgd) [inline distribution/lib/Standard/Database/0.2.28-SNAPSHOT/THIRD-PARTY/org.apache.httpcomponents.httpcore-4.4.13/NOTICES]: Same issue as above.

**Review by radeusgd — CHANGES_REQUESTED** [review]
Code looks good, but there is an issue with license review that needs to be fixed.

**Review by radeusgd — APPROVED** [review]
Looks good to me, assuming that the test suite has at least been ran locally with success.

**Review by radeusgd — APPROVED** [review]

**Review by 4e6 — APPROVED** [review]

**Pull request merged**